From 8f5e4d885f9d4fff15ac4cf09b9c101ec729787b Mon Sep 17 00:00:00 2001
From: Md Sadre Alam <quic_mdalam@quicinc.com>
Date: Tue, 12 Dec 2023 20:55:55 +0530
Subject: [PATCH 480/500] crypto: qce - Fix compilation issue for 32-bit

In 32-bit struct tasklet_struct done_tasklet;
was throwing error error: field 'done_tasklet'
has incomplete type. to fix this add the header
file <linux/interrupt.h> in driver/crypto/qce/core.h
file.

Change-Id: Ie017d0e1de91773564eea08777f98d65dfa5ab07
Signed-off-by: Md Sadre Alam <quic_mdalam@quicinc.com>
---
 drivers/crypto/qce/core.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/crypto/qce/core.h b/drivers/crypto/qce/core.h
index d2c1d0900911..2a0756c1859a 100644
--- a/drivers/crypto/qce/core.h
+++ b/drivers/crypto/qce/core.h
@@ -6,6 +6,7 @@
 #ifndef _CORE_H_
 #define _CORE_H_
 
+#include <linux/interrupt.h>
 #include "dma.h"
 
 #define DEBUG_MAX_RW_BUF 2048
-- 
2.34.1

